content/uiux/concepts/visual-design/visual-design.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: 'Visual Design'
+Description: 'Visual design is a design discipline that aims to develop a strategic aesthetic for digital products.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+  - 'design'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Visual design** is a design discipline that aims to develop a strategic aesthetic for digital products. This aesthetic must organize **[visual design elements](https://www.codecademy.com/resources/docs/uiux/visual-design-elements)** such as line, shape, texture, space, form, and typography into a cohesive whole and support usability. Additionally, visual designers seek to create interfaces that are visually appealing.
+
+Visual design is derived from **graphic design** and **[UI design](https://www.codecademy.com/resources/docs/uiux/ui-design)**. In the digital space, graphic designers may work on visuals related to a product’s marketing and branding. By contrast, visual designers and UI designers are responsible for the look of the product interface. Visual designers focus on the overall aesthetic of a product and may be in charge of building and maintaining a brand’s **[design system](https://www.codecademy.com/resources/docs/uiux/design-system)**.
```

Comment:
```suggestion
Visual design is derived from graphic design and [UI design](https://www.codecademy.com/resources/docs/uiux/ui-design). In the digital space, graphic designers may work on visuals related to a product’s marketing and branding. By contrast, visual designers and UI designers are responsible for the look of the product interface. Visual designers focus on the overall aesthetic of a product and may be in charge of building and maintaining a brand’s [design system](https://www.codecademy.com/resources/docs/uiux/design-system).
```

Reply:
I think this one is ok, graphic design and UI design are unlikely to be new terms for learners.